quadratic-core/src/controller/execution/execute_operation/execute_offsets.rs

```
@@ -286,16 +320,30 @@ impl GridController {
                         })
                         .collect(),
                 });
-            let old_size = sheet.offsets.set_default_width(size);
+
             transaction
                 .reverse_operations
                 .push(Operation::DefaultColumnSize {
                     sheet_id,
                     size: old_size,
                 });
-        } else {
-            return;
         }
+    }
+
+    pub fn execute_default_row_size(
+        &mut self,
+        transaction: &mut PendingTransaction,
+        op: Operation,
+    ) {
+        unwrap_op!(let DefaultRowSize { sheet_id, size } = op);
+
+        let Some(sheet) = self.try_sheet_mut(sheet_id) else {
```

This feels like an anti-pattern (I know, it's everywhere).  Consider returning a result here since we have some mechanism for capturing the result in other exec files.

---

we want this to fail silently, this is being done differently in different fns, needs a comprehensive refactor to make it uniform